contrib/neon/pagestore_smgr.c

```diff
@@ -84,11 +84,6 @@ static char *hexdump_page(char *page);
 
 const int	SmgrTrace = DEBUG5;
 
-/*
- * Pseudo block number used to associate LSN with relation metadata (relation size)
- */
-#define REL_METADATA_PSEUDO_BLOCKNO InvalidBlockNumber
-
```

Comment:
You can move this back here

Reply:
Unfortunately - not, it is use in pagestore_smgr.c in `zenith_get_request_lsn` -> `GeLastWrittenLsn`.
Certainly it is possible to add two versions of `GetLastWrittenLsn` function and somehow  explain `zenith_get_request_lsn` which version to call. But it complicates code and I am not sure that it makes it ore readable. So I left it as it is.